examples/model_compress/pruning/transformers/transformer_pruning.py

```diff
@@ -316,13 +316,11 @@ def forward_runner(model):
         "sparsity": args.sparsity,
         "op_types": ["Linear"],
         "op_names": [x for layer in attention_name_groups[:6] for x in layer]
-    },
-        {
-            "sparsity": args.sparsity / 2,
-            "op_types": ["Linear"],
-            "op_names": [x for layer in attention_name_groups[6:] for x in layer]
-        }
-    ]
+    }, {
```

Comment:
https://nni.readthedocs.io/en/latest/Compression/Pruner.html#transformer-head-pruner 
I mean the example in the documentation here has the wrong indent

Reply:
fixed